runtime/codert_vm/CMakeLists.txt

```diff
@@ -59,7 +59,22 @@ elseif(OMR_ARCH_POWER)
 		pnathelp.s
 	)
 elseif(OMR_ARCH_S390)
-	j9vm_gen_asm(znathelp.m4)
+	set(m4_defines )
+	if(OMR_OS_ZOS)
+		if(OMR_ENV_DATA64)
+			list(APPEND m4_defines  TR_64Bit TR_HOST_64BIT)
+		endif()
+		# NOTE: no flags needed for 31 bit
+
+		if(J9VM_JIT_32BIT_USES64BIT_REGISTERS)
+			list(APPEND m4_defines J9VM_JIT_32BIT_USES64BIT_REGISTERS)
+		endif()
+
+		if(J9VM_JIT_FREE_SYSTEM_STACK_POINTER)
+			list(APPEND m4_defines J9VM_JIT_FREE_SYSTEM_STACK_POINTER)
+		endif()
```

Comment:
Can you point at the equivalent code in the UMA build files?

Reply:
https://github.com/eclipse/openj9/blob/9acbeb8d0bb24638068c7ff0903ec7e8bfe1379e/runtime/makelib/targets.mk.zos.inc.ftl#L100